scripts/cnode-helper-scripts/cntools.sh

```diff
@@ -2306,50 +2309,83 @@ function main {
               fi
               tput rc && tput ed
             fi
+            current_epoch=$(getEpoch)
             getPoolID ${pool_name}
             if [[ ${CNTOOLS_MODE} = "OFFLINE" ]]; then
-              [[ -f "${POOL_FOLDER}/${pool_name}/${POOL_REGCERT_FILENAME}" ]] && pool_registered="YES" || pool_registered="NO"
-              [[ -f "${POOL_FOLDER}/${pool_name}/${POOL_DEREGCERT_FILENAME}" ]] && ledger_retiring="?" || ledger_retiring=""
-            else
+              isPoolRegistered ${pool_name}
```

Comment:
I wonder if this should work in OFFLINE mode at all, as I assume offline mode should mean no internet connection at all

Reply:
True, the offline way of checking if the pool is registered is to look for a pool reg cert, but I guess most only use the transaction -> sign option or maybe wallet list to get wallet address when in offline mode.